custom_model_runner/datarobot_drum/resource/components/Python/r_fit/fit.R

```diff
@@ -24,6 +24,69 @@ init <- function(code_dir) {
 }
 
 
+load_data <- function(input_filename){
+    tmp = readChar(input_filename, file.info(input_filename)$size)
+    data <- read.csv(text=gsub("\r","", tmp, fixed=TRUE))
+    data
+}
+
+
+process_data <- function(input_filename, target_filename, target_name, num_rows){
+    set.seed(1)
+    # read X
+    df <- load_data(input_filename)
+    # set num_rows
+    if (num_rows == 'ALL'){
+        num_rows <- nrow(df)
+    } else {
+        num_rows <- as.integer(a)
+    }
+    # handle target
+    if (!is.null(target_filename) || !is.null(target_name)){
```

Comment:
I love how this code is just like, an absolute port of the python code.

Reply:
I mean....I'm happy to try and get it to be more R-ish, but also I'm not 100% sure what making it more Rish even looks like?

Reply:
maybe @yakov-g  has some thoughts?